cirq/ops/common_gates.py

```diff
@@ -37,6 +38,33 @@
 from cirq.type_workarounds import NotImplementedType
 
 
+def SWAP(*args, **kwargs):
```

Comment:
This is only a half-working shim. For example, `type(cirq.ops.common_gates.SWAP)` will return something different.

I recommend just setting `from cirq.ops.swap_gate import SWAP` with a compatibility note. We will figure out how to deprecate it later.

Reply:
done